### PR TITLE
Enable processing of GEO and KAGRA triggers in PyGRB

### PR DESCRIPTION
**Pull request by a-r-williamson**
These changes will enable the post-processing of GEO and KAGRA triggers within PyGRB

### CODE REVIEW

**Comment by a-r-williamson** [comment]
Successfully tested after 2b816447a0e8b1d033299bb3151d645941603d8f with GEO triggers.
[Example here.](https://ligo.gravity.cf.ac.uk/~andrew.williamson/O3B/pygrb/GRB200415A/GRB200415A_CLOSED/summary.html)

File: pylal/coh_PTF_pyutils.py [hunk 1]
```diff
@@ -78,19 +78,23 @@ def get_bestnr( trig, q=4.0, n=3.0, null_thresh=(4.25,6), snr_threshold=6.,\
...
             i = ifo.lower()
+        elif ifo.lower()[0] == 'k':
+            i = 't'
         else:
             i = ifo[0].lower()
         sens[ifo] = getattr(trig, 'sigmasq_%s' % i.lower()) * \
                         sum(numpy.array([fPlus[ifo], fCross[ifo]])**2)
     ifos.sort(key=lambda ifo: sens[ifo], reverse=True)
     if len(ifos) > 1:
-        for i in xrange(0, 2):
```
Comment (pannarale) [inline pylal/coh_PTF_pyutils.py:89]: You do not want to remove the line `for i in xrange(0, 2):` as the single IFO SNR cut needs to be applied to the two most sensitive detectors (`ifos` is sorted by `sens` above).

The confusing thing here is the double use of `i`.  The loop `for i in xrange(0, 2):` will keep giving you 2 iterations regardless of the `i = ` statements below.  You can try

```
for i in xrange(0,2):
    i = 'a'
    print i
```

to see that.  For clarity, perhaps, change the `i` in `for i in xrange(0, 2):`  into something else.

**Review by pannarale — CHANGES_REQUESTED** [review]

File: pylal/coh_PTF_pyutils.py (hunk 1 above)
Comment (a-r-williamson) [reply pylal/coh_PTF_pyutils.py:89]: Yes, I see. Then shouldn't it just be...
```
for ifo in ifos[:2]:
```

**Review by pannarale — APPROVED** [review]

File: pylal/coh_PTF_pyutils.py (hunk 1 above)
Comment (pannarale) [reply pylal/coh_PTF_pyutils.py:89]: Nice!

**Pull request merged**